src/global/Styles.js

```diff
@@ -604,6 +604,29 @@ const styles = {
     ...defaultTextStyles.body,
     color: '#58595B',
     flex: 1,
+    textAlignVertical: 'center',
+  },
+
+  listItemSubtitle: {
+    ...defaultTextStyles.footnote,
+    color: '#444444',
+    flex: 1,
+    textAlignVertical: 'center',
+  },
+
+  listItemTextContainer: {
+    flex: 1,
+    alignItems: 'stretch',
+    justifyContent: 'space-between',
+    paddingBottom: 16,
+    paddingTop: 12,
+  },
+
+  chatTitleStyle: {
+    ...defaultTextStyles.body,
+    color: '#444444',
+    fontSize: 14,
+    fontWeight: '500',
```

Comment:
can the `color` attribute be taken from a global state by adding it to the global `Colors` variable? As it is repeated.